src/main/java/com/careerfair/q/service/student/request/StudentRequest.java

```diff
@@ -0,0 +1,44 @@
+package com.careerfair.q.service.student.request;
+
+import com.careerfair.q.service.database.deserializer.TimestampDeserializer;
+import com.careerfair.q.util.enums.Role;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.cloud.Timestamp;
+import lombok.Getter;
+
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class StudentRequest {
```

Comment:
Should have private fields with getter and setter

Reply:
should work with requests, will make this change